css/projects_table.css

```diff
@@ -0,0 +1,37 @@
+* {
+    box-sizing: border-box;
+}
+.table table {
```

Comment:
the use of  the same name for class and for tag is a bit confusing to the reader, please consider renaming the class.